packages/plugins/package.json

```diff
@@ -37,6 +37,7 @@
     "version": "pnpm version"
   },
   "dependencies": {
+    "@glint/core": "file:.yalc/@glint/core",
```

Comment:
I'm assuming the dependency is just temporary? We likely want this to be a peerDep because rehearsal it self should install this into the developer's project.

Reply:
yea I can move this to peer/devDeps before we merge

Reply:
moved